src/schema_registry/telemetry/meter.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (c) 2024 Aiven Ltd
+See LICENSE for details
+"""
+
+from dependency_injector.wiring import inject, Provide
+from karapace.config import Config
+from karapace.container import KarapaceContainer
+from opentelemetry import metrics
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import OTLPMetricExporter
+from opentelemetry.sdk.metrics.export import (
+    ConsoleMetricExporter,
+    MetricExporter,
+    MetricReader,
+    PeriodicExportingMetricReader,
+)
+
+
+class Meter:
+    @staticmethod
+    @inject
+    def get_meter(config: Config = Provide[KarapaceContainer.config]) -> metrics.Meter:
+        return metrics.get_meter_provider().get_meter(f"{config.tags.app}.meter")
+
+    @staticmethod
+    @inject
+    def get_metric_reader(config: Config = Provide[KarapaceContainer.config]) -> MetricReader:
+        exporter: MetricExporter = ConsoleMetricExporter()
```

Comment:
@jjaakola-aiven if you remember the recent issue with the systemd logging, maybe we should also implement a `noop` exporter for the metrics, I can check to see if we can reuse the one added for the traces.

The rest of the code can be reviewed.